### PR TITLE
added the ability for admins to see if a user is pending, active or inactive.

### PR DESCRIPTION
**Pull request by sohilpandya**
Admin can now view if the user is either `active`, `inactive` or `pending` in status as seen in the screenshot below. 
screen shot 2017-05-19 at 16 49 50

related to #798

### CODE REVIEW

File: test/unit/isUserAccountActivated.test.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+var isUserAccountActivated = require('../../templates/helpers/isUserAccountActivated.js');
+
+// first array builds dummy request, second string is expected outcome, third is test comment
+var tests = [
+  [[true, true, "active"], true, 'user has an active account'],
+  [[false, null, "inactive"], true, 'user account is inactive'],
+  [[true, false, "pending"], true, 'user account is pending']
+];
+
+// cancel url function
```
Comment (SimonLab) [inline test/unit/isUserAccountActivated.test.js]: wrong comment

**Comment by sohilpandya** [comment]
@SimonLab updated the comments 👍

File: test/unit/isUserAccountActivated.test.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  [[false, null, "inactive"], true, 'user account is inactive'],
+  [[true, false, "pending"], true, 'user account is pending']
+];
+
+// testing each outcome and its results
+tape('check what the account status is for a user', function (t) {
+  tests.forEach(function (test) {
+    var actual = isUserAccountActivated(test[0][0], test[0][1], test[0][2])
+    var expected = test[1];
+    t.equal(actual, expected, test[1])
```
Comment (Jbarget) [inline test/unit/isUserAccountActivated.test.js]: is the last argument supposed to be `test[2]`?

**Comment by SimonLab** [comment]
Thanks for the test update @nelsonic . Looks good.

**Pull request merged**